qiita_pet/templates/study_ajax/add_prep_template.html

```diff
@@ -0,0 +1,119 @@
+<script type="text/javascript" src="/static/vendor/js/jquery.validate.min.js"></script>
+<script type="text/javascript>">
+$(document).ready(function () {
+  // Hide the user and new ontology divs
+  $("#user-ena-info").hide();
+  $("#new-ena-info").hide();
+  // If the ena-ontology select changes, check if we need to show the user list
+  $("#ena-ontology").change(function ( event ) {
+      if( $("#ena-ontology").val() == "Other" ) { $("#user-ena-info").show(); }
```

Comment:
This block is over indented for two spaces. Also, any reason why the whole thing is in a line? Most notably for the else block? same comment applies to the rest of the file, would make more sense to have them in their own line.